src/lib/components/toggle/Toggle.component.js

```diff
@@ -18,78 +18,82 @@ const ToggleContainer = styled.div`
 const Switch = styled.label`
   position: relative;
   display: inline-block;
-  width: 24px;
+  width: 42px;
```

Comment:
The target size of the toggle itself is 24px*14px (width * height).

With 42px width, the label is way too small compared to the toggle.